requirements.txt

```diff
@@ -50,5 +50,5 @@ importlib-metadata==3.10.1 #pinned to fix the pytest deprecation warning: Select
 nplusone==0.8.0
 pytest==8.0.2
 pytest-cov==4.1.0
-setuptools==68.0.0 #pin to remediate snyk ReDoS#5477
+setuptools==70.3.0 #pin to remediate snyk vulnerability 
```

Comment:
@tmpayton  A later version of setuptools v71.1.0 is now available. Would you try to upgrade to this versions and see how it goes? FYI, CMS is pinned to v71.1.0 during 7/30 deployment.

Reply:
@pkfec Thank you for the review! I tried upgrading to v71 but there is an error with marshmallow pagination. [Here](https://app.circleci.com/pipelines/github/fecgov/openFEC/2872/workflows/8466319b-2565-484f-b810-66a33f0b7482/jobs/5682) is the circleci build. And I also created this ticket [here](https://github.com/fecgov/marshmallow-pagination/issues/18) for the fix.

Reply:
@tmpayton Thanks for the update!